docs/rancher.md

```diff
@@ -0,0 +1,34 @@
+# Deploying Antrea on Rancher Managed Cluster
+
+<!-- toc -->
+- [Deploy Rancher Server](#deploy-rancher-server)
+- [Deploy Antrea Using Rancher UI](#deploy-antrea-using-rancher-ui)
+<!-- /toc -->
+
+## Deploy Rancher Server
+
+Follow these [steps](https://rancher.com/docs/rancher/v1.6/en/installing-rancher/installing-server/) to deploy rancher server ui dashboard.
+
+## Deploy Antrea Using Rancher UI
+
+* Sign-in to the UI dashboard, and switch to cluster management.
+
+![rancher_cluster_management](https://raw.githubusercontent.com/antrea-io/antrea/main/docs/assets/rancher_cluster_management.png)
+
+* Create new Node template there under the RKE configuration section.
+
+![rancher_node_template](https://raw.githubusercontent.com/antrea-io/antrea/main/docs/assets/rancher_node_template.png)
+
+* After saving the template, proceed with creating new cluster, under the clusters tab click on create button, then enter the name, and number of worker and controlplane Nodes respectively.
+
+![rancher_create](https://raw.githubusercontent.com/antrea-io/antrea/main/docs/assets/rancher_create.png)
+
+* click on “Edit as YAML", and set the network plugin to null, and add a addon for antrea, in the following manner:
+addons_include:
+  -<link of the antrea.yml file>
```

Comment:
here the user can use any yml file, I have not specified the exact link here because then the user might be of the thought that we need to use this particular yml only, that's why I have mentioned it like this to make it more general.